.travis.yml

```diff
@@ -10,8 +10,12 @@ matrix:
     env: "DB=sqlite SERVER_BRANCH=stable17"
   - php: 7.3
     env: "DB=sqlite SERVER_BRANCH=v17.0.0beta4"
+  - php: 7.3
+    env: "DB=pgsql"
   - php: 7.2
     env: "DB=mysql"
+  - php: 7.2
```

Comment:
IMO one postgres job is sufficient. It's highly unlikely that php7.2 works with postgres if 7.3 doesn't or the other way around

Reply:
Sure I will change that, also I see there is an issue with the command postgres --version in the last travis CI run that needs to be fixed.